src/Graphs/partitionedgraphs/partitionedgraph.jl

```diff
@@ -94,12 +94,26 @@ function edges(pg::PartitionedGraph, partitionedges::Vector{<:PartitionEdge})
   return unique(reduce(vcat, [edges(pg, pe) for pe in partitionedges]))
 end
 
+function boundary_partitionedges(
+  pg::PartitionedGraph, partitionvertices::Vector{<:PartitionVertex}; kwargs...
+)
+  return PartitionEdge.(
+    boundary_edges(partitioned_graph(pg), parent.(partitionvertices); kwargs...)
+  )
+end
+
+function boundary_partitionedges(
+  pg::PartitionedGraph, partitionvertex::PartitionVertex; kwargs...
+)
+  return boundary_partitionedges(pg, PartitionVertex[partitionvertex]; kwargs...)
```

Comment:
```suggestion
  return boundary_partitionedges(pg, [partitionvertex]; kwargs...)
```
Julia should already infer the type is `Vector{<:PartitionVertex}`.